src/qml/qgismobileapp.qml

```diff
@@ -2116,6 +2117,48 @@ ApplicationWindow {
         return result + padding * 2;
     }
 
+    Row {
+      bottomPadding: parent.topMargin
+
+      QfToolButton {
+        icon.source: Theme.getThemeVectorIcon( "ic_undo" )
```

Comment:
The icons here should be named following the following standard: ic_{blah}_{black,white}_{24,48,etc.}. In your case it'd be ic_{undo,redo}_black_24dp